pkg/network/network.go

```diff
@@ -286,3 +285,77 @@ func cleanUpNamespaceLabels(ctx context.Context, client k8sclient.Client) []erro
 
 	return []error{}
 }
+
+func cnaoKNMStateObjects(operandNamespace string) ([]*unstructured.Unstructured, error) {
+	objects := []runtime.Object{
+		&v1.ServiceAccount{
+			ObjectMeta: metav1.ObjectMeta{Namespace: operandNamespace, Name: "nmstate-handler"},
+		},
+		&v1.Service{
+			ObjectMeta: metav1.ObjectMeta{Namespace: operandNamespace, Name: "nmstate-webhook"},
+		},
+		&appsv1.DaemonSet{
+			ObjectMeta: metav1.ObjectMeta{Namespace: operandNamespace, Name: "nmstate-handler"},
+		},
+		&appsv1.Deployment{
+			ObjectMeta: metav1.ObjectMeta{Namespace: operandNamespace, Name: "nmstate-webhook"},
+		},
+		&appsv1.Deployment{
+			ObjectMeta: metav1.ObjectMeta{Namespace: operandNamespace, Name: "nmstate-cert-manager"},
+		},
+		&rbacv1.Role{
+			ObjectMeta: metav1.ObjectMeta{Namespace: operandNamespace, Name: "nmstate-handler"},
+		},
+		&rbacv1.RoleBinding{
+			ObjectMeta: metav1.ObjectMeta{Namespace: operandNamespace, Name: "nmstate-handler"},
+		},
+		&rbacv1.ClusterRole{
+			ObjectMeta: metav1.ObjectMeta{Name: "nmstate-handler"},
```

Comment:
how do we know that we are in fact catching old nmstate `ClusterRole` & `ClusterRoleBinding` here? couldn't this also catch (and later delete) standalone knmstate ones?

Reply:
CNAO only deletes owned objects with this PR https://github.com/kubevirt/cluster-network-addons-operator/pull/1231/files#diff-e93827340ebe6b239ff60427dba9f2881d96e7bba9a8a9b07a09cb64582dd73fR113-R116

Reply:
ah so it's taken care off. thnx!